### PR TITLE
Decompile asm/daycare.s

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/daycare.c [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
+IWRAM_DATA u16 gUnknown_030004D8[4];
+IWRAM_DATA u16 gUnknown_030004E0[4];
+IWRAM_DATA u16 gUnknown_030004E8[12];
+IWRAM_DATA u16 gUnknown_03000500[4];
+
+static void sub_80417F4(struct DayCareMail *);
+static void sub_80420FC(struct Pokemon *, u16, struct DayCareData *);
+static u8 daycare_relationship_score(struct DayCareData *);
+
+const u8 *const gUnknown_08209AC4[] = {
```
Comment (huderlem) [inline src/daycare.c]: This deserves a descriptive label.  Something like gDaycareMonStatusTexts

Comment (PikalaxALT) [reply src/daycare.c]: yeah, i should probably go through and rename everything

File: src/daycare.c [hunk 2]
```diff
@@ -71,275 +106,108 @@ s8 Daycare_FindEmptySpot(struct BoxPokemon * daycare_data)
...
 
-    for(i = 0; i < 100; i++){
-        if(TryIncrementMonLevel(mon) == FALSE) goto end;
-
-        r6 = 1;
-        while((temp = MonTryLearningNewMove(mon, r6)) != 0){
-            r6 = 0;
-            if(temp == 0xffff){
-                DeleteFirstMoveAndGiveMoveToMon(mon, gMoveToLearn);
+    for (i = 0; i < 100; i++)
```
Comment (huderlem) [inline src/daycare.c]: MAX_LEVEL

File: src/record_mixing.c [hunk 3]
```diff
@@ -29,7 +29,7 @@ extern u8 ewram[];
 #define unk_2018000 (*(struct PlayerRecords *)(ewram + 0x18000))
 #define unk_2008000 (*(struct PlayerRecords *)(ewram + 0x08000))
 
-extern struct RecordMixing_UnknownStruct gUnknown_02038738;
+extern struct RecordMixingDayCareMail gUnknown_02038738;
```
Comment (huderlem) [inline src/record_mixing.c]: unnecessary, since it's defined in global.h?

File: src/battle_anim_80CA710.c [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+// Includes
+#include "global.h"
+
+// Static type declarations
+
+// Static RAM declarations
+
+IWRAM_DATA u32 filler_03000724;
```
Comment (huderlem) [inline src/battle_anim_80CA710.c]: This `filler_03000724` seems unused

**Pull request merged**